lib/instrumentation/azure-functions.js

```diff
@@ -376,7 +376,9 @@ function instrument(agent) {
         direction: context?.options?.trigger?.direction,
       });
       // Output bindings
-      bindingDefinitions.push(context?.options?.return);
+      if(context?.options?.return) {
```

Comment:
```suggestion
      if (context?.options?.return) {
```

lint fix. I tried to push this fix to your branch, but that failed on permissions:

```
% git push wladimirguerra HEAD
Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 11 threads
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 652 bytes | 163.00 KiB/s, done.
Total 5 (delta 4), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
To github.com:wladimirguerra/apm-agent-nodejs.git
 ! [remote rejected]   HEAD -> wladimirguerra/main (permission denied)
error: failed to push some refs to 'github.com:wladimirguerra/apm-agent-nodejs.git'
```

Not sure why. Perhaps because it is on your "main" branch, or could be I'm just messing up the git-fu.